app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditFragmentPresenter.kt

```diff
@@ -151,6 +153,10 @@ class ProfileEditFragmentPresenter @Inject constructor(
         fragment,
         Observer {
           if (it is AsyncResult.Success) {
+            snackbarManager.showSnackbar(
```

Comment:
You should add a test for snackbar being shown after profle has been deleted in `ProfileEditFragmentTest`

Reply:
okay sure